facts/3.14/ubuntu-22.04-x86_64.facts

```diff
@@ -514,118 +525,108 @@
   "osfamily": "Debian",
   "partitions": {
     "/dev/loop0": {
-      "backing_file": "/var/lib/snapd/snaps/core20_1494.snap",
+      "backing_file": "/var/lib/snapd/snaps/core20_2264.snap",
       "filesystem": "squashfs",
-      "mount": "/snap/core20/1494",
-      "size": "61.92 MiB",
-      "size_bytes": 64925696
+      "mount": "/snap/core20/2264",
+      "size": "63.95 MiB",
+      "size_bytes": 67051520
     },
     "/dev/loop1": {
-      "backing_file": "/var/lib/snapd/snaps/snapd_15904.snap",
+      "backing_file": "/var/lib/snapd/snaps/lxd_28373.snap",
       "filesystem": "squashfs",
-      "mount": "/snap/snapd/15904",
-      "size": "44.70 MiB",
-      "size_bytes": 46870528
+      "mount": "/snap/lxd/28373",
+      "size": "87.03 MiB",
+      "size_bytes": 91258880
     },
     "/dev/loop2": {
-      "backing_file": "/var/lib/snapd/snaps/lxd_22923.snap",
+      "backing_file": "/var/lib/snapd/snaps/snapd_21465.snap",
       "filesystem": "squashfs",
-      "mount": "/snap/lxd/22923",
-      "size": "79.95 MiB",
-      "size_bytes": 83832832
+      "mount": "/snap/snapd/21465",
+      "size": "38.73 MiB",
+      "size_bytes": 40615936
     },
     "/dev/sda1": {
       "filesystem": "ext4",
       "label": "cloudimg-rootfs",
       "mount": "/",
-      "partuuid": "5de372cc-01",
+      "partuuid": "3a5d506e-01",
       "size": "40.00 GiB",
       "size_bytes": 42948607488,
-      "uuid": "427c026c-8396-43e2-8152-2a3383ffd337"
+      "uuid": "e62a0045-72ea-439a-9034-30c57d243a14"
     }
   },
   "path": "/opt/puppetlabs/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin",
   "physicalprocessorcount": 1,
-  "processor0": "Intel(R) Core(TM) i7-9800X CPU @ 3.80GHz",
-  "processor1": "Intel(R) Core(TM) i7-9800X CPU @ 3.80GHz",
+  "processor0": "AMD Ryzen 7 PRO 4750U with Radeon Graphics",
+  "processor1": "AMD Ryzen 7 PRO 4750U with Radeon Graphics",
   "processorcount": 2,
   "processors": {
     "count": 2,
     "isa": "x86_64",
     "models": [
-      "Intel(R) Core(TM) i7-9800X CPU @ 3.80GHz",
-      "Intel(R) Core(TM) i7-9800X CPU @ 3.80GHz"
+      "AMD Ryzen 7 PRO 4750U with Radeon Graphics",
+      "AMD Ryzen 7 PRO 4750U with Radeon Graphics"
     ],
     "physicalcount": 1
   },
   "productname": "VirtualBox",
-  "puppetversion": "5.5.22",
+  "puppetversion": "6.28.0",
```

Comment:
is that even legal? :D

Reply:
the previous 5.5.22 was wrong. puppet 5.5.22 was installed and then the facts generated with facter 3.14 from rubygems.org.